src/include/cell_inline.h

```diff
@@ -845,6 +848,19 @@ __wt_cell_unpack_safe(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CE
         break;
     }
 
+    /*
+     * Unpack any fast-truncate information. Note that there is no way to write fast-truncate
+     * information to disk in versions before 6.1, but this information may still exist in the
+     * database files if we are downgrading from newer versions.
```

Comment:
I think this deserves another sentence along the lines of "If the information is present, skip past it and ignore the values. That is the equivalent of assuming the content is globally visible, which matches the existing behavior in 6.1."

Reply:
@agorrod I don't quite understand the second part of your sentence when you say it matches the existing behavior in 6.1. Do you mean that by ignoring the fast truncate information in 6.0, and is equivalent to how 6.1 treats globally visible truncates?

Reply:
Oh, I think I just had my versions wrong. I meant ignoring version information is equivalent to the version that doesn't ever include visibility when writing out truncated content.

Reply:
@agorrod I've updated the comment in [20f851a](https://github.com/wiredtiger/wiredtiger/pull/9396/commits/20f851a5776322e2b1d9ea3e082dcca4b9f94742), let me know if you'd like me to make any more changes to the wording.